### DISCUSSION TITLE
Does the UI Kit need a paid Figma license?

### DISCUSSION THREAD

**Discussion by JuncalLP** [primefaces/community#3029] [Figma UI Kit]
New to the world of Figma, thinking about acquiring the UI Kit since it's on sale and it seems like it would greatly improve my DX. My intention is to be able to visually edit design tokens and easily export them to be used in my web apps. Two doubts:

1. Is this (among) the purposes of the UI Kit or am I under a wrong assumption?
2. I need a paid Figma license to use the UI Kit or I'm good with the free license? 

Thanks in advance.

**Comment by onursenture** [comment]
Hi, With the UI Kit, you can visually edit components and modify tokens for your Figma designs. However, at the moment, there isn't a solution to export these changes for use in your web app. We are working on this feature, but it still needs to be completed. Additionally, please be aware that the plugin or API that will handle this export task will be a separate product and will be sold separately.

As far as I remember, the Free Figma tier has no restrictions on the use of files like UI kit. However, since you can't publish the UI Kit as a library, if you plan to import it across multiple files, you may need to upgrade to a paid Figma tier.
